cmd/skopeo/layers.go

```diff
@@ -18,7 +19,12 @@ var layersCmd = cli.Command{
 		if err != nil {
 			return err
 		}
-		src := image.FromSource(rawSource)
+		src := image.FromSource(rawSource, []string{
+			// TODO: skopeo layers only support these now
+			// eventually we'll remove this command altogether...
+			manifest.DockerV2Schema1SignedMIMEType,
+			manifest.DockerV2Schema1MIMEType,
+		})
```

Comment:
Yeah, sure, but that is not a rationale for this change to `layers.go` nor does it explain how “skopeo layers only support these now”.

Or are you saying that the code would work fine but you want to intentionally restrict it to v2s1 to force users to use `skopeo copy`?